lib/types/universal.d.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * @fileoverview typings for "eslint/universal" module
+ * @author 唯然<weiran.zsd@outlook.com>
+ */
+
+export type { Linter } from "./index.d.ts";
```

Comment:
I'm getting an error "'Linter' cannot be used as a value because it was exported using 'export type'." when using `Linter` from `eslint/universal` in a .ts file:

![image](https://github.com/user-attachments/assets/34361d68-69e3-46b5-bac6-371366532ea8)